config/routes.rb

```diff
@@ -69,6 +69,7 @@
   
   get 'series/bulk', to: 'series#bulk_new'
   post 'series/bulk', to: 'series#bulk_create'
+  get 'series/stale', to: 'series#stale'
```

Comment:
Oh, I get to try this comment-per-line thing again :=) So.... if RoR is all about convention-over-configuration, why is this routing instruction needed at all? Doesn't it seem uselessly redundant?